public/app/features/provisioning/RepositoryStatusPage.tsx

```diff
@@ -129,3 +181,82 @@ function FilesTable({ name }: FilesTableProps) {
     </Stack>
   );
 }
+
```

Comment:
@Clarity-89 -- I could not figure out how to make the query work with RTK, so I'll leave that as an exercise for when you get back :)